src/components/appRouter.js

```diff
@@ -666,7 +669,8 @@ define(['loading', 'globalize', 'events', 'viewManager', 'layoutManager', 'skinM
 
     function pushState(state, title, url) {
         state.navigate = false;
-        page.pushState(state, title, url);
+        history.pushState(state, title, url);
```

Comment:
where is `history` defined?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/History/pushState